src/foam/nanos/script/jShell/InstructionPresentation.java

```diff
@@ -24,17 +24,30 @@ public InstructionPresentation(JShell jShell) {
   }
 
   public List<String> parseToInstruction(List<String> scripts) {
-    int i = 0;
+    int    i           = 0;
     String codeToParse = "";
     while ( i < scripts.size() ) {
-      codeToParse += scripts.get(i);
-      SourceCodeAnalysis.CompletionInfo info = jShell.sourceCodeAnalysis().analyzeCompletion(codeToParse);
-      if ( info.completeness().isComplete() ) {
+      String instraction = checkEmptyInstraction(scripts.get(i));
```

Comment:
Is "instraction" a word? What does it mean? Do you mean "instruction"?

Reply:
instruction